### PR TITLE
Noah items

### PR DESCRIPTION
**Pull request by NoahBartell**
added my items

### CODE REVIEW

**Review by instMeyFTW — APPROVED** [review]

File: dunwoody_disaster/CharacterFactory.py [hunk 1]
```diff
@@ -345,16 +345,34 @@ def Noah() -> Character:
...
+        char.defenses = [
+            Item.Armor(
+                name="PR REJECTED",
+                armorVal=5,
+                staminaCost=5,
+                magicCost=5,
+                magicDefense=0,
+            ),
+            Item.Armor(
+                name="Heavy Metal",
```
Comment (chanson02) [inline dunwoody_disaster/CharacterFactory.py:369]: Are you a [metal enjoyer](https://i.redd.it/hwstw5mw39m71.jpg)?

File: dunwoody_disaster/CharacterFactory.py [hunk 2]
```diff
@@ -345,16 +345,34 @@ def Noah() -> Character:
...
-        # ]
+        char.weapons = [
+            Item.Weapon(name="Power Chord", damage=5, magic=5, stamina=5, magical=True),
+            Item.Weapon(
+                name="600 Hour Scrum Meeting",
+                damage=1,
+                magic=1,
+                stamina=1,
+                magical=False,
+            ),
```
Comment (chanson02) [inline dunwoody_disaster/CharacterFactory.py]: I feel like that would take a lot more stamina lol

**Review by chanson02 — APPROVED** [review]
Hilarious, keep up the good work.

File: dunwoody_disaster/CharacterFactory.py (hunk 1 above)
Comment (NoahBartell) [reply dunwoody_disaster/CharacterFactory.py:369]: Maybe..

**Pull request merged**